mitodl/open-discussions
# Support confirming email on a different device/browser

#### What are the relevant tickets?
Fixes #905 

#### What's this PR do?
Adds `partial_token` to the confirmation email link so the process can be resumed on a browser/device other than the one it was begun on.

#### How should this be manually tested?

- Go to `/signup` and enter an email
- Copy/paste the confirmation link to an incognito window and verify it works
